client/acct_mgr.h

```diff
@@ -169,12 +169,11 @@ struct ACCT_MGR_OP: public GUI_HTTP_OP {
     int parse(FILE*);
     virtual void handle_reply(int http_op_retval);
 
-    ACCT_MGR_OP(GUI_HTTP* p) {
+    ACCT_MGR_OP(GUI_HTTP* p) : global_prefs_xml("") {
```

Comment:
This is a question of style.
I see no particular purpose in this change

Reply:
It’s more optimized to use initialization list for constructors. Of course the gain is micro perf but why letting it away